website_logo/__manifest__.py

```diff
@@ -1,26 +1,24 @@
 # Copyright 2015 Agile Business Group - Lorenzo Battistini
 # Copyright 2016 Tecnativa - Antonio Espinosa
 # Copyright 2017 Tecnativa - David Vidal
+# Copyright 2018 eslaAmer - Eslam Amer
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 {
     'name': 'Website logo',
     'summary': 'Website company logo',
-    'version': '11.0.1.0.0',
-    'category': 'Website',
-    'author': "Agile Business Group, "
+    'version': '12.0.1.0.0',
+    'category': 'website',
+    'author': "EslAmer, "
```

Comment:
This migration hasn't promoted you to co-author. Please put only on contributors

Reply:
done